src/calendareventlistmodel.h

```diff
@@ -44,12 +44,14 @@ class CalendarEventListModel : public QAbstractListModel, public QQmlParserStatu
     Q_PROPERTY(int count READ count NOTIFY countChanged)
     Q_PROPERTY(QStringList identifiers READ identifiers WRITE setIdentifiers NOTIFY identifiersChanged)
     Q_PROPERTY(QStringList missingItems READ missingItems NOTIFY missingItemsChanged)
+    Q_PROPERTY(bool loading READ loading NOTIFY loadingChanged)
 
 public:
     enum EventListRoles {
         EventObjectRole = Qt::UserRole,
         OccurrenceObjectRole,
-        IdentifierRole
+        IdentifierRole,
+        ChildRole // Used by classes inheriting CalendarEventListModel
```

Comment:
Think more often the name of the last item in enum is having the word Last in rather than Child. Alternatively could just do something like EventObjectRole +100 on the inherited classes first item to define some number range which should suffice here.

Reply:
Ok, I went for `LastRole`. Thanks for the suggestion.